demo/apisix/conf/config.yaml

```diff
@@ -5,4 +5,4 @@ deployment:
 
 ext-plugin:
   path_for_test: /tmp/runner.sock
-  cmd: ['java', '-jar', '-Xmx1g', '-Xms1g', '/usr/local/apisix/token_revoker_plugin.jar']
\ No newline at end of file
+  cmd: ['java', '-jar', '-DREDIS_HOST=redis.example.com', '-DREDIS_PORT=6379', '-Xmx1g', '-Xms1g', '/usr/local/apisix/token_revoker_plugin.jar']
```

Comment:
How about the Redis authentication?
E.g Redis password was needed on our preprod:
```
redisURL=redis://${env:OP_REDIS_PASSWORD}@${env:OP_REDIS_URL}:${env:OP_REDIS_PORT}
```